cypress/e2e/auth.cy.ts

```diff
@@ -15,7 +15,8 @@ describe("Authentication tests", () => {
     it("Sign out", () => {
         cy.login();
 
-        cy.get("input[type=submit]").contains("Sign out").click();
+        cy.get("button[aria-label='Profile Settings']").eq(1).click();
+        cy.get("li").contains("Sign Out").click({ force: true });
```

Comment:
assuming this `force: true` is still necessary... that's a shame (and a little weird) - can we do anything like `.should("be.visible")` instead?